index.html

```diff
@@ -280,16 +280,15 @@ <h2>Keep in touch</h2>
     </div>
   </div>
 
-  <div class="footer">
+  <div class="footer">       
```

Comment:
Unnecesary whitespace change

Reply:
weird I had 2 checkboxes near prettier when I uploaded.

Reply:
Guess it doesn't style html